src/fixtures/x.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  pathPrefix: "hello",
```

Comment:
More consistent styling:

```suggestion
  pathPrefix: 'hello'
```

Reply:
That's my secret "TDD" fixture. It got check in by accident 🙃